FindMe/View Controllers/LoginViewController.m

```diff
@@ -7,7 +7,7 @@
 //
 
 #import "LoginViewController.h"
-#import <Parse/Parse.h>
+#import "DatabaseManager.h"
```

Comment:
Way better than importing parse. (technically you are still importing parse but this makes that easier to change later)